skopt/optimizer/optimizer.py

```diff
@@ -467,7 +471,7 @@ def tell(self, x, y, fit=True):
                 # minimization starts from `n_restarts_optimizer` different
                 # points and the best minimum is used
                 elif self.acq_optimizer == "lbfgs":
-                    x0 = X[np.argsort(values)[:self.n_restarts_optimizer]]
+                    x0 = X[np.argsort(values)]
```

Comment:
Hmm, actually, I remember now. We did this on purpose. The idea was to do a very coarse optimization with many random points. Then start LBFGS from the best points. This was to mimick the implementation of Spearmint. 

In the end, I would rather leave this unchanged.